docs/openshift/faq.md

```diff
@@ -0,0 +1,9 @@
+# FAQ (OpenShift Specific)
+
+This FAQ is reserved for questions specifically pertaining to using OLM on OpenShift.
+
+## Developing an Operator
+
+### Q: I'm developing a new version of an operator already published to one of the default catalogs (redhat-operators, certified-operators, or community-operators). How can I hide the published versions from OLM and the OperatorHub UI?
+
+Deleting the default `OperatorSource` and `CatalogSource` of the catalog your operator exists in will prevent OLM and the OperatorHub UI from using it. To delete a default `OperatorSource`, you must first [create or update the `marketplace-operator's` configuration](operator-marketplace.md#configuration) to prevent it from recreating them.
```

Comment:
Is it valuable to also mention that this only the case for openshift 4.2 +?

Reply:
Because 4.1 requires marking those resources as unmanaged by the cvo.

Reply:
Ah, I didn't know that. Thanks! I'll update this soon.

Reply:
updated